src/szenarien/SzenarienConst.js

```diff
@@ -28,11 +28,18 @@ const SZENARIEN = [
     "getAnalyse": getAnalyseForErstVsZweit,
   },
   {
-    "id": "aenderung",
-    "titel": "Größte Änderung",
+    "id": "gewinner",
+    "titel": "Größter Gewinner",
     "untertitel": "Welche Partei hat am meisten zugelegt?",
-    "beschreibung": "Die Wahlbezirke sind in der Farbe der Partei eingefärbt, die im Vergleich zur letzten Bundestagswahl absolut betrachtet die größten Stimmgewinne hat.",
-    "getAnalyse": getAnalyseGroessteAenderung,
+    "beschreibung": "Die Wahlbezirke sind in der Farbe der Partei eingefärbt, die im Vergleich zur letzten Bundestagswahl relativ betrachtet die größten Stimmgewinne hat.",
```

Comment:
In der Beschreibung sollten wir klar machen, dass es um die Zweitstimmen geht: "...  die im Vergleich zur Bundestagswahl 2013 die meisten Zweitstimmen hinzugewonnen hat."

Analog für die Verlierer-Ansicht.